docs/modes-of-operation.rst

```diff
@@ -326,3 +326,56 @@ of 99 seconds and the specified string. TXT Records with names starting
 with ``_tstamp.`` get their value (rdata) set to the current time stamp.
 A records are appended with a TXT record. All other records are
 unhandled.
+
+.. _modes-of-operation-axfrend:
+
+Signaling end of an AXFR transfer for specific zone(s) using a script
+---------------------------------------------------------------------
+
+The PowerDNS Authoritative Server can invoke a Lua script on the of
+incoming AXFR zone transfer. The user-defined function ``axfr_end(zone)``
+within your script is invoked for each zone that has LUA-AXFR-END-SCRIPT
+defined in the ``domainmetadata``.
```

Comment:
Could use a reference here to the metadata docs.

Reply:
this commits adds reference to domainmetadata docs: https://github.com/m0rcq/pdns/commit/d40ec81ce9baaadcf5b883dde576f463426ccf69